code/datums/wires/conveyor.dm

```diff
@@ -15,5 +15,5 @@
 /datum/wires/conveyor/interactable(mob/user)
 	if(!..())
 		return FALSE
-	fingerman = user
+	fingerman = WEAKREF(user)
```

Comment:
you're gonna need to call `resolve()` on the fingerman ref to get it to spit out the mob it's holding for you

Reply:
wait but arent I just assigning it there?

Reply:
sorry not there, where it's used in that file.